examples/asgardeo-integration/Ballerina.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+org = "wso2"
+name = "asgardeo_integration"
+version = "0.1.0"
+distribution = "2201.12.7"
+
+[build-options]
+observabilityIncluded = true
+
+[[dependency]]
+org = "ballerinax"
+name = "scim"
+version = "1.0.0"
+repository="local"
```

Comment:
```suggestion
```

Reply:
Fixed in [bcdef0f](https://github.com/ballerina-platform/module-ballerinax-scim/pull/20/commits/bcdef0fef6334fba1716297267d5d9569a315f1b)